packages/geolocator/tizen/src/setting.cc

```diff
@@ -9,6 +9,8 @@
 #include <package_manager.h>
 
 namespace {
+constexpr char kSettingAppId[] = "com.samsung.clocksetting.apps";
```

Comment:
https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace

Blank lines immediately inside a declaration of a namespace improves readability.